src/finufft.cpp

```diff
@@ -185,16 +156,16 @@ void set_nhg_type3(FLT S, FLT X, finufft_opts opts, finufft_spread_opts spopts,
   else
     Ssafe = max(Ssafe, 1 / X);
   // use the safe X and S...
-  FLT nfd = 2.0 * opts.upsampfac * Ssafe * Xsafe / PI + nss;
+  FLT nfd = FLT(2.0 * opts.upsampfac * Ssafe * Xsafe / PI + nss);
```

Comment:
`const auto`

Reply:
`auto` is OK, `const` doesn't work, since `nfd` may change on the next line.